blocks/init/src/Blocks/custom/map/assets/utils.js

```diff
@@ -15,6 +16,11 @@ import OLZoom from 'ol/control/Zoom.js';
 
 import { applyStyle as OLMBStyleApply } from 'ol-mapbox-style';
 
+import { Style, Fill, Stroke, Icon } from 'ol/style';
+
+// eslint-disable-next-line max-len
+const mapMarkerImage = 'data:image/svg+xml,<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M16 7.35938C16 11.9873 11.8347 16.9857 10.1728 18.7958C9.80508 19.1964 9.19492 19.1964 8.82717 18.7958C7.16529 16.9857 3 11.9873 3 7.35938C3 3.84719 5.91015 1 9.5 1C13.0899 1 16 3.84719 16 7.35938Z" fill="%233A66A8" stroke="%232E5590" stroke-linejoin="round"/><path d="M9.5 9.25C10.4665 9.25 11.25 8.4665 11.25 7.5C11.25 6.5335 10.4665 5.75 9.5 5.75C8.5335 5.75 7.75 6.5335 7.75 7.5C7.75 8.4665 8.5335 9.25 9.5 9.25Z" fill="white"/></svg>';
```

Comment:
Is this the same as the one in the `map-components.js`? If so, it would be better to have it in the manifest, yes?

Reply:
Won't work, can't import manifest in front JS.